README.md

```diff
@@ -92,6 +95,28 @@ class LoginForm extends Form
     {
         return 'LoginForm';
     }
+
+    /** Add rules */
+    public function rules(): array
+    {
+        return [
+            'login' => $this->loginRules()
+        ];
+    }
+
+    /** Define login rules */
+    private function loginRules(): array
+    {
+        return [
+            new Required(),
+            (new HasLength())
+            ->min(4)
+            ->max(40)
+            ->tooShortMessage('Is to short.')
```

Comment:
should read `'Is too short.'`, also below and in tests

Reply:
done.